src/modules/home/marqueeBanner.tsx

```diff
@@ -28,10 +37,25 @@ const Banner = () => {
 const MarqueeContent = ({ ariaHidden }: { ariaHidden?: boolean }) => {
   return (
     <div
-      className="marquee flex w-full shrink-0 items-center justify-evenly gap-6 text-xl"
+      className="marquee flex w-full shrink-0 items-center gap-0 text-xl"
       aria-hidden={ariaHidden}
     >
-      <Banner />
+      <Swiper
+        modules={[Navigation, Pagination, Scrollbar, A11y, Autoplay]}
+        spaceBetween={50}
+        slidesPerView={1}
+        loop={true}
+        pagination={{ clickable: true }}
+        className="hero-slider h-full"
+        autoplay={{ delay: 5000 }}
+      >
+        <div className="flex items-center gap-4 whitespace-nowrap">
```

Comment:
Not sure why do you need this. We already have the Marquee scrolling.

Are these really making any effect?
``` 
modules={[Navigation, Pagination, Scrollbar, A11y, Autoplay]}
        spaceBetween={50}
        slidesPerView={1}
        loop={true}
        pagination={{ clickable: true }}
        className="hero-slider h-full"
        autoplay={{ delay: 5000 }}
```
They were configured to customize for the slider in hero section.